src/Geom_Line.f90

```diff
@@ -359,7 +359,7 @@ ELEMENTAL FUNCTION distance_LineType_to_PointType(line,p) RESULT(d2)
   CLASS(LineType),INTENT(IN) :: line
   TYPE(PointType),INTENT(IN) :: p
   REAL(SRK) :: d2
-  d2=-1._SRK
+  d2=-HUGE(d2)
```

Comment:
This is probably unecessary. The reason I initially added this is that the routine can generate a negative value by accident, contrary to the description. That being said I believe the magnitude of that value is guaranteed to be quite low.